src/script/acquisition-sdk.ts

```diff
@@ -80,7 +82,21 @@ export class AcquisitionManager {
         this._ignoreAppVersion = configuration.ignoreAppVersion;
     }
 
+    private isRecoverable = (statusCode: number): boolean => statusCode >= 500 || statusCode === 408 || statusCode === 429;
```

Comment:
You check this when response code is not 200 - what about other 2xx or 3xx responses - are they recoverable or not?